packages/flutter/test/widgets/platform_view_test.dart

```diff
@@ -160,4 +162,191 @@ void main() {
         ])
     );
   });
+
+  testWidgets('Android view gets touch events', (WidgetTester tester) async {
+    final int currentViewId = platformViewsRegistry.getNextPlatformViewId();
+    final FakePlatformViewsController viewsController = new FakePlatformViewsController(TargetPlatform.android);
+    viewsController.registerViewType('webview');
+    await tester.pumpWidget(
+        const Align(
```

Comment:
some indent issues

Reply:
Did you mean indent with 2 spaces? not sure why this is what I get from the intellij auto formatter.
Dartfmt does something similar (4 spaces then 2):

```dart
  testWidgets('Android view gets touch events', (WidgetTester tester) async {
    final int currentViewId = platformViewsRegistry.getNextPlatformViewId();
    final FakePlatformViewsController viewsController =
        new FakePlatformViewsController(TargetPlatform.android);
    viewsController.registerViewType('webview');
    await tester.pumpWidget(const Align(
        alignment: Alignment.topLeft,
        child: SizedBox(
          width: 200.0,
          height: 100.0,
          child: AndroidView(viewType: 'webview'),
        )));

    final TestGesture gesture =
        await tester.startGesture(const Offset(50.0, 50.0));
    await gesture.up();

    expect(
        viewsController.motionEvents[currentViewId + 1],
        orderedEquals(<FakeMotionEvent>[
          const FakeMotionEvent(AndroidViewController.kActionDown, <int>[0],
              <Offset>[Offset(50.0, 50.0)]),
          const FakeMotionEvent(AndroidViewController.kActionUp, <int>[0],
              <Offset>[Offset(50.0, 50.0)]),
        ]));
  });

```

Reply:
I think dartfmt does something really different depending on whether you put a comma after the last parameter. I don't have a strong opinion